cordova-plugin-outline/android/java/org/outline/log/SentryErrorReporter.java

```diff
@@ -112,6 +116,15 @@ public static void init(Context context, final String dsn) {
     }
   }
 
+  private static boolean isVpnServiceContext(Context context) {
+    try {
+      return context.getClass().getName().contains("vpn");
+    } catch (Exception e) {
+      Log.w(SentryErrorReporter.class.getName(), "Failed to determine running context");
```

Comment:
Should we return true in this case to be safe?

Reply:
I was thinking it would be safer to assume that we're in the main app process so as not to limit user feedback reports.

Reply:
could we add an explicit case for being in org.outline.adroid.client?  It makes more sense ot be explicit here and save us a potential future headache

Reply:
not sure I follow - if the check fails, we fall back to the current behavior of not sampling.

Reply:
I was thinking about a future where something about app architectures change, but I think I'm being overly cautious.  LGTM